packages/flutter/test/material/app_bar_test.dart

```diff
@@ -3167,6 +3167,37 @@ void main() {
     expect(titleOffset.dx, backButtonOffset.dx + titleSpacing);
   });
 
+  testWidgets('AppBar actions padding can be adjusted', (WidgetTester tester) async {
+    final Key appBarKey = UniqueKey();
+    final Key actionKey = UniqueKey();
+
+    Widget buildAppBar({ EdgeInsetsGeometry? actionsPadding }) {
+      return MaterialApp(
+        home: Scaffold(
+          appBar: AppBar(
+            key: appBarKey,
+            actions: <Widget>[
+              SizedBox.square(key: actionKey, dimension: 40.0),
+            ],
+            actionsPadding: actionsPadding,
+          ),
+        ),
+      );
+    }
+
+    await tester.pumpWidget(buildAppBar());
+
+    // Test actions position without actions padding.
+    Offset actionsOffset = tester.getTopRight(find.byKey(actionKey));
+    final Offset appBarOffset = tester.getTopRight(find.byKey(appBarKey));
+    expect(actionsOffset.dx, appBarOffset.dx);
+
+    const EdgeInsets actionsPadding = EdgeInsets.only(right: 8.0);
+    await tester.pumpWidget(buildAppBar(actionsPadding: actionsPadding));
+    actionsOffset = tester.getTopRight(find.byKey(actionKey));
+    expect(actionsOffset.dx, equals(appBarOffset.dx - actionsPadding.right));
```

Comment:
```suggestion
    expect(appBarOffset.dx - actionsPadding.right, actionsOffset.dx);
```
Proposed change to make this expect clause similar to the prebious one.

Reply:
I feel the original way is good enough since the first parameter for `expect()` is `actual` which looks better match `actionsOffset.dx`